apps/api/.env.example

```
@@ -1,4 +1,5 @@
 NODE_ENV=development
+API_PORT=service-port
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The API_PORT key should go before the NODE_ENV key

(UnorderedKey)